.github/workflows/check-configs-changes.yml

```diff
@@ -0,0 +1,70 @@
+name: "Check Config Changes"
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+
+jobs:
+  check-configs-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Install Dependencies
+        run: |
+          sudo apt update
+          sudo apt install -y make
```

Comment:
Are you sure we need to install it explicitly? https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#installed-apt-packages

Also looks like we are not even using `make` in this action? Is it a leftover from testing?